src/IEvent.ts

```diff
@@ -1,13 +1,23 @@
-// import Player from "./Player";
+export interface IEvent {
```

Comment:
shouldn't we change the name for this file to be more descriptive?

Reply:
Would just `src/Event.ts` be good?

Reply:
sure